manifests/resource-topology-exporter-ds.yaml

```diff
@@ -69,6 +69,8 @@ spec:
             mountPath: "/host-etc"
           - mountPath: /etc/sriov-config
             name: my-sriov-config-vol
+          - name: host-var
+            mountPath: "/host-var"
```

Comment:
let's narrow down as much as possible the subtree we mount, especially if this needs to be mounted in RW